superset_patchup/utils.py

```diff
@@ -44,3 +44,12 @@ def is_safe_url(target_url):
     test_url = urlparse(urljoin(request.host_url, target_url))
     return test_url.scheme in ("http",
                                "https") and ref_url.netloc == test_url.netloc
+
+
+def is_valid_provider(user_input, static_provider):
```

Comment:
How do you feel about adding [type hints](https://docs.python.org/3/library/typing.html) here?

Reply:
ill add them, just a sec.